src/v1_TeslaMateAPICarsCharges.go

```diff
@@ -179,6 +179,11 @@ func TeslaMateAPICarsChargesV1(c *gin.Context) {
 	if err != nil {
 		log.Fatal(err)
 	}
+	
+	// if no errors, but ChargeData is empty, return a valid response with an empty set
+	if len(ChargeData) == 0 {
```

Comment:
line 184: should be `ChargesData` and not `ChargeData`

Reply:
Fixed!